src/test/java/org/takes/http/BkParallelTest.java

```diff
@@ -23,15 +23,108 @@
  */
 package org.takes.http;
 
+import com.jcabi.http.request.JdkRequest;
+import com.jcabi.http.response.RestResponse;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.takes.Request;
+import org.takes.Response;
+import org.takes.Take;
+import org.takes.tk.TkEmpty;
+
 /**
  * Test case for {@link BkParallel}.
  *
  * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
  * @version $Id$
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @since 0.15.2
- * @todo #220:30min/DEV This unit test not implemented but has to be.
- *  E.g. you could pass your own a ExecutorService implementation to the ctor
- *  and verify execute() calls.  But feel free to try another way.
  */
-public class BkParallelTest {
+@SuppressWarnings(
+    {
+        "PMD.DoNotUseThreads",
+        "PMD.CyclomaticComplexity",
+        "PMD.AvoidInstantiatingObjectsInLoops"
+    }
+)
+public final class BkParallelTest {
+    /**
+     * BkParallel runs requests in parallel, hence even
+     * when handling of a request blocks, other requests
+     * should be handled.
+     * @throws java.lang.Exception If some problem inside
```

Comment:
@komaz No need to use fully qualified name of `java.lang.Exception`, let's shorten it and use `Exception` instead.